app/src/main/res/layout/fragment_settings_safe.xml

```diff
@@ -106,6 +106,13 @@
                     android:background="@drawable/background_selectable_white"
                     app:setting_openable="false" />
 
+                <View
+                    android:id="@+id/master_copy_separator"
+                    android:layout_width="match_parent"
+                    android:layout_height="2dp"
```

Comment:
Is this double size? Should we use `@dimen/item_separator_height`?

Reply:
I missed one. This should also be `@dimen/item_separator_height` which is 2dp

Reply:
Fixed in dfe45c2bdd12532d7f1091d8eb32a29fde1ef724